junit-jupiter-params/src/main/java/org/junit/jupiter/params/converter/DefaultArgumentConverter.java

```diff
@@ -185,18 +190,23 @@ private Object valueOf(Class targetType, String source) {
 
 	private static class StringToJavaTimeConverter implements StringToObjectConverter {
 
-		private static final Map<Class<?>, Function<CharSequence, ?>> CONVERTERS;
```

Comment:
Thanks for fixing this left-over from earlier days.

iirc, we first supported `CharSequence` instead of `String` and there was a reason why we ended up with using `String`.